### ISSUE TITLE
Brackets become unresponsive during `ember s` command

### ISSUE THREAD

**Issue by FezVrasta** [adobe/brackets#11777]
I develop with Ember and recently I've started with `ember-cli` and Ember.js 2.

Everytime I run in a console `ember s` to serve the project, Brackets becomes unresponsive and takes ages to perform any action.

I've tested it only on OS X Yosemite and El Capitan, with both the stable release of Brackets and with the new 1.6 with the latest Bracket-shell compiled by sources

**Comment by petetnt** [comment]
Ember does tons of temporarily files in the `tmp`-folder while serving / building, which in return takes a ton of juice from file watchers and other file related functions. The solution is to filter the folder out with something like [gruehle/exclude-folders](https://github.com/gruehle/exclude-folders) which should bring back the performance.

**Comment by FezVrasta** [comment]
Yup it works :)

Probably Brackets should have some auto detection for folders with lot of files edited inside with a warning that says that such folder will being disabled from the watchers.

Just like it happens right now with folders with weird javascript inside.

**Comment by petetnt** [comment]
Agreed with the warning / some sort of detection.

For reference, a random `ember` project I have open right now has a whopping `10595` temporary files and `3613` folders generated in mere 3 days.

**Comment by FezVrasta** [comment]
@abose I don't think that this issue should be closed, the problem is real and we should find a reliable solution

**Comment by abose** [comment]
Reopening as we need to unwatch volatile folders.
